memgpt/main.py

```diff
@@ -34,9 +34,7 @@
 app.command(name="configure")(configure)
 app.command(name="list")(list)
 app.command(name="add")(add)
-app.command(name="server")(server)

```

Comment:
inadvertent delete? will patch back

Reply:
Yeah sorry, accidentally deleted a few things during the cherry pick.